routersploit/validators.py

```diff
@@ -21,6 +21,18 @@ def address(addr):
     return addr.netloc or addr.path
 
 
+def enum(valid_values):
```

Comment:
Great idea and perfect execution. 👏 

I would change name to `choice` to be more in line with various existing APIs e.g. `random.choice()` or ORMs' choice fields.

It would be nice  if you add few tests in `routersploit.test.test_validators.ValidatorsTest`